crates/bitwarden-ssh/src/lib.rs

```diff
@@ -15,22 +15,22 @@ uniffi::setup_scaffolding!();
 #[derive(Serialize, Deserialize, Debug)]
 #[cfg_attr(feature = "wasm", derive(Tsify), tsify(into_wasm_abi, from_wasm_abi))]
 #[cfg_attr(feature = "uniffi", derive(uniffi::Record))]
-pub struct SshKey {
+pub struct UnencryptedSshKey {
     /// The private key in OpenSSH format
     pub private_key: String,
     pub public_key: String,
     pub key_fingerprint: String,
 }
```

Comment:
Can we not just use `SshKeyView` here?

Reply:
Does that not mean that we would have to move the impl `impl TryFrom<PrivateKey> for SshKeyView` to `bitwarden-vault` & thus making `bitwarden-vault` dependent on some SSH dependencies for parsing, plus requiring adding conversion specific errors to `bitwarden-vault` (currently `SshKeyExportError::KeyConversionError`)?

Reply:
Yes, you do lose access to using `TryFrom`, but we could just implement it as a regular function, or have a wrapper NewType that we unwrap from before returning it.